contracts/0.0.1/group.js

```diff
@@ -10438,11 +10453,23 @@ ${this.getErrorInfo()}`;
         }),
         process({ data, meta, contractID }, { state, getters }) {
           memberLeaves({ username: meta.username, dateLeft: meta.createdDate }, { meta, state, getters });
+          meta.isRemoveOurselves = true;
```

Comment:
Pls let me know if this(mutating the structure of the meta object for sending down an information to `sideEffect()` of `gi/contracts/group/removeMember`) is an issue. Or if you have an advice regarding this.